tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Spark.ipynb

```diff
@@ -0,0 +1,1498 @@
+   "id": "12d40c77",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# this will work \n",
+    "data_context.assistants.onboarding.run(\n",
+    "          batch_request=multi_batch_batch_request,\n",
+    "          categorical_columns_rule = {\n",
+    "              'cardinality_limit_mode': 'VERY_MANY',\n",
+    "             #'mostly': 1.0,\n",
+    "             #'strict_min': False, \n",
+    "             #'strict_max': False, \n",
+    "             #'false_positive_rate': 0.05, \n",
+    "             #'estimator': 'bootstrap', \n",
+    "             #'n_resamples': 9999,\n",
+    "             #'random_seed': None, \n",
+    "             #'quantile_statistic_interpolation_method': 'nearest', \n",
+    "             #'quantile_bias_correction': False, \n",
+    "             #'quantile_bias_std_error_ratio_threshold': None,\n",
+    "             #'include_estimator_samples_histogram_in_details': False, \n",
+    "             #'truncate_values': {\n",
+    "             #    'lower_bound': 0.0,\n",
+    "             #   'upper_bound': None\n",
+    "             #}, \n",
+    "             #'round_decimals': 15\n",
+    "          }\n",
+    "    )\n",
+    "\n",
+    "# and so will this \n",
+    "data_context.assistants.onboarding.run(\n",
+    "          batch_request=multi_batch_batch_request,\n",
+    "          categorical_columns_rule = {\n",
+    "              'cardinality_limit_mode': 'VERY_MANY',\n",
+    "          }\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "788358ff",
+   "metadata": {},
+   "source": [
+    "# Alternate configurations of `batch_spec_passthrough` "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35b04f19",
+   "metadata": {},
+   "source": [
+    "As mentioned in the previous section, options for `spark.read()` function, can be passed in through the `batch_spec_passthrough` parameter.\n",
+    "\n",
+    "The `batch_spec_passthrough` parameter can either be defined at the\n",
+    "\n",
+    "- `DataConnector`-level\n",
+    "- `Asset`-level\n",
+    "- `BatchRequest`-level.\n",
+    "\n",
+    "And can be used to pass in `reader_method` (csv), or `reader_options` like (`header = True`)\n",
+    "\n",
+    "```python\n",
+    "\"batch_spec_passthrough\":{\n",
+    "    \"reader_method\": \"csv\",\n",
+    "    \"reader_options\":{\n",
+    "        \"header\": True,\n",
+    "        \"schema\": schema # the schema we defined earlier\n",
+    "    }\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f0d1045",
+   "metadata": {},
+   "source": [
+    "When loading in a `Datasource` configuration through ` data_context.add_datasource(**datasource_config)`, we need to serialize the `schema` by calling the `jsonValue()` function of the `StructType` object [More information here](https://spark.apache.org/docs/3.1.3/api/python/reference/api/pyspark.sql.types.StructType.html#pyspark.sql.types.StructType.jsonValue)\n"
```

Comment:
Could you put a period before and after `More information here`.

Reply:
Done!